source/includes/examples/_energyplus.html.md

```diff
@@ -2,7 +2,7 @@
 
 > ![EnergyPlus Graph](../images/energyplus-graph.png)
 
-[**Download the full source code on GitHub**][1] so you can run it yourself on the Bonsai Platform.
+[**Download the full source code on GitHub**][1] if you want to run this simulator locally. If you want to run Cart Pole remotely on the Bonsai Platform as a managed simulator, create a new BRAIN selecting the Cart Pole demo on [beta.bons.ai][4].
```

Comment:
Sorry to send this back again, but it looks like there is a copy & paste error here now. `Cart Pole` should be `EnergyPlus`